packages/moonstone/VideoPlayer/VideoPlayer.js

```diff
@@ -1694,8 +1694,7 @@ const VideoPlayerBase = class extends React.Component {
 		Spotlight.setPointerMode(false);
 
 		if (this.focusDefaultMediaControl()) {
-			ev.preventDefault();
-			ev.stopPropagation();
+			ev.nativeEvent.stopImmediatePropagation();
```

Comment:
`ev` is not a native event and will throw an error.

Reply:
You are right! It used to receive the keyboard event but doesn't any longer. I'll need to revisit this functionality.